product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx

```diff
@@ -72,37 +72,41 @@ Set the following environment variables in `/opt/cdcreader/run-cdcreader.sh` wit
 #export QUARKUS_LOG_CATEGORY__COM_ENTERPRISEDB__LEVEL=DEBUG
 ```
 
-## Running the reader
-
-```shell
-cd /opt/cdcreader
-./run-cdcreader.sh
-```
-
 ### Parameters explanation
 
 #### DBZ_ID
-ID to identify the reader. Users can define the ID they think will be useful with below constraints:
 
-1. ID length should lower or equal to 255
-2. ID can only contain lowercase characters, uppercase characters, numbers, underscore(_) and hyphen(-)
-3. ID should be unique, two reader instances should not use the same ID
+This is the name you assign to a migration to identify it. 
```

Comment:
This DBZ_ID is assigned to a reader not a migration

Reply:
Is this also the name that will later appear as the source name in the EDB Cloud Service (BigAnimal) UI? If yes, we could say: 
```suggestion
This is the name you assign to identify a reader. This name will later appear as a _source_ in the EDB Cloud Service UI. 
```
If not, we can simply say: 
```suggestion
This is the name you assign to a reader.
```